android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -271,10 +271,18 @@ private void setUiColor(int color) {
         toolbar.setBackgroundColor(color);
         collapsingToolbarLayout.setBackgroundColor(color);
         collapsingToolbarLayout.setContentScrimColor(darkColor);
-        Views.changeGlowColor(color,scrollView);
 
-        if(Views.isCompatible(Build.VERSION_CODES.LOLLIPOP))
+        if (Views.isCompatible(Build.VERSION_CODES.LOLLIPOP)) {
             getWindow().setStatusBarColor(darkColor);
+            Views.setEdgeGlowColorScrollView(color, scrollView);
+            speakersRecyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
+                @Override
+                public void onScrollStateChanged(RecyclerView recyclerView, int newState) {
+                    super.onScrollStateChanged(recyclerView, newState);
+                    Views.setEdgeGlowColorRecyclerView(speakersRecyclerView, color);
+                }
+            });
+        }
 
```

Comment:
does it seem necessary here? Over tracks session activity it looks great but over here didnt look good to me. let others review.